docs/paper/dev/api/how-do-plugins-work.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: /dev/how-do-plugins-work
+description: How plugins work in Paper.
+---
+
+# How Plugins Work
+
+## Introduction
+
+Plugins are a way to extend the functionality of a Minecraft server. They are written in JVM-based language such as 
+Java, Kotlin, Groovy or Scala and can be loaded and unloaded at runtime. Plugins are loaded from the `plugins` folder 
+in the server directory. Plugins will be loaded from a `.jar` file. 
+
+## Plugin Lifecycle
+
+Plugins are loaded and unloaded at runtime. When a plugin is loaded, it is initialized and enabled. When a plugin is 
+unloaded, it is disabled and finalized.
+
+### Initialization
+
+When a plugin is loaded, it is initialized. This means that the plugin is loaded into memory and its `onLoad` 
+method is called. This method is used to initialize the plugin and set up any resources that it needs. Most of the
+Bukkit API is not available at this point, so it is not safe to interact with it.
+
+### Enabling
+
+When a plugin is enabled, its `onEnable` method is called. This method is used to set up any resources that the plugin 
+needs to run. This method is called when the plugin is initialized but before the server has started ticking, so it is 
+safe to register event listeners and other resources that the plugin needs to run, however often not safe to interact 
+with a lot of APIs.
+
+This is when you can also open database connections, start threads, and other things that are not safe to do in the
+`onLoad` method.
+
+### Disabling
+
+When a plugin is disabled, its `onDisable` method is called. This method is used to clean up any resources that the 
+plugin has allocated. This method is called before all plugins are unloaded, and is meant for any cleanup that needs to
+be done before the plugin is unloaded. This may include saving data to disk or closing connections to databases.
+
+## Event Listeners
+
+Events are a way for plugins to listen to things that happen in the server. Events are fired when something happens,
```

Comment:
`Events are a way for plugins to listen to things that happen in the server.`
`Events are fired when something happens, and plugins can listen to these events and run code when they are fired.`
`Plugins can listen to this event and run code when it is fired.`

the same thing (roughly) is repeated a few times in one paragraph